inst/shiny/DiagnosticsExplorer/server.R

```diff
@@ -4119,69 +2722,226 @@ shiny::shinyServer(function(input, output, session) {
+        "databaseId",
+        "databaseName",
+        "vocabularyVersionCdm",
+        "vocabularyVersion",
+        "description",
+        "startTime",
+        "runTime",
+        "runTimeUnits",
+        "sourceReleaseDate",
+        "cdmVersion",
+        "cdmReleaseDate" ,
+        "observationPeriodMinDate",
+        "observationPeriodMaxDate"
       )
     )
-    sketch <- htmltools::withTags(table(class = "display",
-                                        thead(
-                                          tr(
-                                            th(rowspan = 2, "ID"),
-                                            th(rowspan = 2, "Name"),
-                                            th(
-                                              "Vocabulary version",
-                                              colspan = 2,
-                                              class = "dt-center",
-                                              style = "border-right:1px solid silver;border-bottom:1px solid silver"
-                                            ),
-                                            th(rowspan = 2, "Description")
-                                          ),
-                                          tr(lapply(
-                                            c("CDM source", "Vocabulary table"), th, style = "border-right:1px solid silver;border-bottom:1px solid silver"
-                                          ))
-                                        )))
-    table <- DT::datatable(
-      data ,
-      options = options,
-      container = sketch,
-      rownames = FALSE,
-      class = "stripe compact"
-    ) 
-    return(table)
-  }, server = TRUE)
+    
+    dataColumns <- c("personsInDatasource",
+                     "recordsInDatasource",
+                     "personDaysInDatasource")
+    
+    data <- getDisplayTableSimple(data = data,
+                                  keyColumns = keyColumns,
+                                  dataColumns = dataColumns)
+    return(data)
+  })
+  
+  #Sign in/Authentication ---------------------------------------------
+  activeLoggedInUser <- reactiveVal(NULL)
+  observeEvent(eventExpr = input$signInModelPopUp,
+               handlerExpr = {
+                 if (is.null(activeLoggedInUser())) {
+                   shiny::showModal(
+                     shiny::modalDialog(
+                       title = "Sign in",
+                       easyClose = TRUE,
+                       size = "s",
+                       footer =  tagList(
+                         shiny::modalButton("Cancel"),
+                         shiny::actionButton("signIn", "Sign in")
+                       ),
+                       tags$div(
+                         shiny::textInput(
+                           inputId = "userName",
+                           label = "User name",
+                           width = NULL
+                         ),
+                         shiny::passwordInput(
```

Comment:
This is bad - we're taking a users password in the shiny app (in plain text) then passing it to the webapi instance.